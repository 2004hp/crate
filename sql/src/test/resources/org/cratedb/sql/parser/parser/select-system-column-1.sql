select * from t1 where "_version"=1