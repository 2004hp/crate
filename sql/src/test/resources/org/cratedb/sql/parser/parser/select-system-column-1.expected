org.cratedb.sql.parser.parser.CursorNode@862ef09
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@173b3c2c
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@31eb789e

		[0]:		
		org.cratedb.sql.parser.parser.AllResultColumn@5a3be6ac
		tableName: null
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@12a5547c

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@244da274
		tableName: t1
		updateOrDelete: null
		null
		correlation Name: null
		null
	whereClause: 		
		org.cratedb.sql.parser.parser.BinaryRelationalOperatorNode@64e4804a
		operator: =
		methodName: equals
		type: null
		leftOperand: 			
			org.cratedb.sql.parser.parser.SystemColumnReference@55a2ae05
			columnName: _version
			tableName: null
			type: null
		rightOperand: 			
			org.cratedb.sql.parser.parser.NumericConstantNode@2b4ef62f
			value: 1
			type: INTEGER NOT NULL
